### PR TITLE
feat: add terraform example usage to apply alerts to Hello app 

### PR DESCRIPTION
**Pull request by sanderblue**
This pull request has been created to demonstrate and provide a reference to a working Terraform example that provisions "golden signal" alerts for the Hello demo application. The Terraform configuration also includes provisioning of a dashboard containing data pertaining to the Hello application.

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+  title             = var.dashboard_name != "" ? var.dashboard_name : var.app_name
+  icon              = "line-chart"
+  grid_column_count = 12
+  visibility        = "all"
+  editable          = "editable_by_all"
+
+  widget {
+    title         = ""
+    visualization = "markdown"
+    source        = "# You've just created your dashboard\n\nAn explanation here of what to do next.\n\nMaybe put a [link here](https://newrelic.com) to point them at a relevant page."
```
Comment (jerelmiller) [inline terraform/main.tf:49]: We should probably get some proper copy for this markdown widget. This was my (poor) attempt at lorem ipsum.

Comment (sanderblue) [reply terraform/main.tf:49]: Agreed. Where would we get this copy from?

Comment (jerelmiller) [reply terraform/main.tf:50]: @djsauble or @markweitzel would you be able to help provide some copy we can use for the markdown widget?

Comment (djsauble) [reply terraform/main.tf:50]: What do we want users to do next once they've created the dashboard?

**Pull request closed without merging**